user/caribou/module/include/CaribouEvent2StdEventConverter.hh

```diff
@@ -60,10 +60,13 @@ namespace eudaq {
     bool Converting(eudaq::EventSPC d1, eudaq::StandardEventSP d2, eudaq::ConfigurationSPC conf) const override;
     static const uint32_t m_id_factory = eudaq::cstr2hash("CariboudSiPMEvent");
   private:
+    uint8_t getQuadrant(const uint16_t& col, const uint16_t& row) const;
```

Comment:
Could even be

```cpp
static uint8_t getQuadrant(const uint16_T& col, const uint16_t& row);
```

since it does not rely on any member variables of `dSiPMEvent2StdEventConverter`. More a style question though

Reply:
> I haven't checked all the calculations of the timestamp, but looks good to me overall.
Inge checked them and asked me to implement the possibility to make the effective number of bits for the fine TDC configurable and quadrant dependent, which I did here. Otherwise she had no complains. I also tested this version and results are the same as before the change, when one uses the default number of effective bits.

> Could even be `static uint8_t getQuadrant(const uint16_T& col, const uint16_t& row);` ...
Changed.